ystring/length.go

```diff
@@ -0,0 +1,31 @@
+package ystring
+
+import (
+	"unicode/utf8"
+)
+
+// Length returns the number of runes in a given string.
+func Length(s string) int {
+	return utf8.RuneCountInString(s)
+}
+
+// Truncate returns first n runes of s.
+func Truncate(s string, n int) string {
+	switch {
+	case n < 0:
+		panic("ystring: negative Truncate length n")
+	case n == 0:
+		return s[0:0]
+	case n >= len(s):
+		return s
+	}
+
+	cnt := 0
+	for idx := range s {
+		if n <= cnt {
+			return s[0:idx]
+		}
+		cnt += 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [should replace cnt += 1 with cnt++](https://app.codacy.com/app/an9an63/gut/pullRequest?prid=4530708)